src/layout.css

```diff
@@ -46,48 +46,67 @@
  */
 
 /**
- * Grids are made by adding the `grid` class to a parent and `col col--{size}` classes to children. Column classes include `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
+ * Grids require at minimum two elements: A parent with the class `grid` and a child with the class `col`. By default, each `col` in a grid will occupy equal space. The `grid` class includes `*-mm`, `*-ml`, and `*-mxl` variations to target screen sizes.
  *
- * Grids and their columns use margin and padding for layout, so you should not apply any other margin or padding to these elements. Instead, apply such styling to parent and child elements, and use these elements for the grid layout only.
+ * When using these classes, do not apply left or right margin or padding directly to the \`grid\` or \`col\` elements or you will face unexpected side-effects.
  *
```

Comment:
In that case, introducing the grid only at `-mm` would also be an acceptable solution: `col-mm` would do the trick. If you needed guaranteed full width at small sizes, `w-full w-auto-mm col-mm` would work. If other people like the feel of `col col--12 col--even-mm` I'd be okay with adding it though